### PR TITLE
[8.0] account_move_line_stock_info

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Account Move Line Stock Move
============================

This module adds the stock move to the account move lines that it generates.

Usage
=====

* The stock manager can check the journal items generated by a stock
  move, by accessing to the picking, then clicking on one of the moves, to
  display the list of journal items.

* An accounting user or manager can review the details of a move that is
  associated to a journal item that she/he is checking.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
cc @andhit-r you might as well me interested in this one.
cc @eficent: your review is welcome!

File: account_move_line_stock_info/tests/test_account_move_line_stock_move.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        # Create Product
+        self.product = self._create_product()
+        # company
+
+        # Create users
+        self.stock_user = self._create_user('stock_user',
+                                            [self.group_stock_user,
+                                             self.group_account_invoice],
+                                            self.company)
+        self.account_invoice = self._create_user('account_invoice',
```
Comment (LoisRForgeFlow) [inline account_move_line_stock_info/tests/test_account_move_line_stock_move.py:62]: Maybe something like 'accountant' or 'account_user' will be more descriptive, do you agree?

File: account_move_line_stock_info/tests/test_account_move_line_stock_move.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        for move in picking_in.move_lines:
+            self.assertEqual(len(move.account_move_line_ids), 2)
+            for aml in move.account_move_line_ids:
+                self.assertEqual(aml.name, move.name)
+                account_move_line = aml
+
+        for move in picking_in.move_lines:
+            self.assertEqual(len(move.account_move_line_ids), 2)
+            for aml in move.account_move_line_ids:
+                self.assertEqual(aml.name, move.name)
```
Comment (LoisRForgeFlow) [inline account_move_line_stock_info/tests/test_account_move_line_stock_move.py]: This for is duplicated, maybe it should be erased.

**Review by LoisRForgeFlow — COMMENTED** [review]
Some minor comments in the code.

**Review by AaronHForgeFlow — COMMENTED** [review]
Same comments as @lreficent

File: account_move_line_stock_info/tests/test_account_move_line_stock_move.py (hunk 1 above)
Comment (JordiBForgeFlow) [reply account_move_line_stock_info/tests/test_account_move_line_stock_move.py:62]: account_invoice is a defined group in odoo.

File: account_move_line_stock_info/tests/test_account_move_line_stock_move.py (hunk 2 above)
Comment (JordiBForgeFlow) [reply account_move_line_stock_info/tests/test_account_move_line_stock_move.py]: OK, fixed. Thanks

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

File: account_move_line_stock_info/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Account Move Line Stock Move",
+    "version": "8.0.1.0.0",
+    "depends": [
+        "stock_account"
+    ],
+    "author": "Eficent,"
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
```
Comment (andhit-r) [inline account_move_line_stock_info/__openerp__.py:12]: Why not using Eficent website?

File: account_move_line_stock_info/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    "category": "Warehouse Management",
+    "installable": True,
+    "license": "AGPL-3",
+    "data": [
+        'security/ir.model.access.csv',
+        'views/account_move_line_view.xml',
+        'views/stock_move_view.xml',
+    ],
+    "images": [],
```
Comment (andhit-r) [inline account_move_line_stock_info/__openerp__.py]: Remove empty dictionary

File: account_move_line_stock_info/__openerp__.py (hunk 3 above)
Comment (JordiBForgeFlow) [reply account_move_line_stock_info/__openerp__.py:12]: In the end it's more practical that people should get access to the OCA repo. We're not maintaing any page in our website that better describes this module.

**Review by andhit-r — APPROVED** [review]
It's harder for accountant to see the stock move. But this due original Odoo view. The only way accountant can access stock move info is via Journal Items menu. Click journal item data so odoo will enter editable mode. And then switch to form view using right-top view switcher button. 

A solution for above problem is a nice to have. But overall :+1:

File: account_move_line_stock_info/__openerp__.py (hunk 4 above)
Comment (JordiBForgeFlow) [reply account_move_line_stock_info/__openerp__.py]: Adding the stock move to the account move line is needed to ensure traceability. It's not necessary to make this field super visible in the journal entries/items.

Another module should implement an "Inventory Audit Report', such as https://help.sap.com/saphelp_sbo882/helpdata/en/45/22e90a73c80108e10000000a114a6b/content.htm

**Pull request merged**